_po/ja/how-to/laravel.po

```diff
@@ -103,38 +134,55 @@ msgid ""
 "DB_PASSWORD=postgres\n"
 "```"
 msgstr ""
+"```ini\n"
+"DB_CONNECTION=pgsql\n"
+"DB_HOST=127.0.0.1\n"
+"DB_PORT=5432\n"
+"DB_DATABASE=pgroonga_laravel\n"
+"DB_USERNAME=postgres\n"
+"DB_PASSWORD=postgres\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.